client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/plan-only.tsx

```diff
@@ -183,10 +189,10 @@ const PlanOnlyThankYou = ( {
 	footerDetails.push( {
 		key: 'footer-support',
 		title: translate( 'Everything you need to know' ),
-		description: translate( 'Explore our support guides and find an answer to every question.' ),
+		description: translate( 'Visit Help Center and find an answer to every question.' ),
```

Comment:
Looks like you forgot to make changes also [here](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/email/google-workspace-set-up-thank-you/index.tsx#L30) or we don't need there?

Reply:
Good catch, thank you